hazelcast/src/test/java/com/hazelcast/test/TestPartitionUtils.java

```diff
@@ -170,17 +161,15 @@ public int getPartitionId() {
 
         @Override
         public void run() {
-            final InternalPartitionService partitionService = node.nodeEngine.getPartitionService();
-            final long[] replicaVersions = partitionService.getPartitionReplicaVersions(partitionId);
-            this.replicaVersions = Arrays.copyOf(replicaVersions, replicaVersions.length);
+            InternalPartitionService partitionService = node.nodeEngine.getPartitionService();
+            long[] replicaVersionsArray = partitionService.getPartitionReplicaVersions(partitionId);
```

Comment:
I prefer the replicaVersions. Array in this case only adds noise; doesn't provide any added value.